src/lib/viewers/controls/media/VolumeControls.scss

```diff
@@ -1,24 +1,30 @@
 @import './styles';
 
 .bp-VolumeControls {
+    position: relative;
     display: flex;
     align-items: center;
 }
 
 .bp-VolumeControls-flyout {
+    display: none;
     flex: 0 0 auto;
     max-width: 0;
     height: $bp-MediaControl-height;
     overflow: hidden;
     transition: max-width 200ms ease-in-out;
 
     &.bp-is-open {
-        max-width: 100px;
+        position: absolute;
+        top: -40px;
+        left: -5px;
+        display: block;
```

Comment:
Can we simplify this by just having `display: block` in this bracket and everything else in `bp-VolumeControls-flyout`?